src/core/namespace.c

```diff
@@ -112,7 +112,7 @@ static const MountEntry protect_kernel_modules_table[] = {
 /* ProtectKernelLogs= option */
 static const MountEntry protect_kernel_logs_table[] = {
         { "/proc/kmsg",          INACCESSIBLE, true },
-        { "/dev/kmesg",          INACCESSIBLE, true },
+        { "/dev/kmsg",           INACCESSIBLE, true },
```

Comment:
we generally prefer merging "perfect" patch sets, i.e. patchsets that are perfectly bisectable. Hence please fold this fixup into the original patch that broke it. Or in other words, in the same PR there shouldn't be patches that that break and then others that fix it.